solver/llbsolver/provenance/predicate.go

```diff
@@ -233,6 +233,7 @@ func FilterArgs(m map[string]string) map[string]string {
 		"cgroup-parent":      {},
 		"image-resolve-mode": {},
 		"platform":           {},
+		"cache-imports":      {},
```

Comment:
Missing `cache-exports`?

Reply:
In follow-up we could look also do smth smarter and detect if fields contain sensistive info, otherwise we keep them.

Reply:
Frontends only have access to imports. Exports are handled directly by buildkit itself.